src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -500,30 +429,34 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 	} else if (cmd_mavlink.command == MAV_CMD_GET_MESSAGE_INTERVAL) {
 		get_message_interval((int)roundf(cmd_mavlink.param1));
 
-	} else if (cmd_mavlink.command == MAV_CMD_REQUEST_FLIGHT_INFORMATION) {
-		send_flight_information();
-
-	} else if (cmd_mavlink.command == MAV_CMD_REQUEST_STORAGE_INFORMATION) {
```

Comment:
We will have to support these commands for backwards compatibility, I suppose.

Reply:
Yes we will. This is why we need microservices versioning and the new command metadata stuff.